### PR TITLE
Added name-only and examples in the kubectl plugin help

### PR DESCRIPTION
**Pull request by Ritikaa96**
Fixes #https://github.com/kubernetes/kubectl/issues/1432

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/plugin/plugin.go [hunk 1]
```diff
@@ -40,15 +40,19 @@ var (
 		Plugins provide extended functionality that is not part of the major command-line distribution.
 		Please refer to the documentation and examples for more information about how write your own plugins.
 
-		The easiest way to discover and install plugins is via the kubernetes sub-project krew.
-		To install krew, visit [krew.sigs.k8s.io](https://krew.sigs.k8s.io/docs/user-guide/setup/install/)`))
+		The easiest way to discover and install plugins is via the kubernetes sub-project krew: [krew.sigs.k8s.io].
+		To install krew, visit https://krew.sigs.k8s.io/docs/user-guide/installing-plugins/`))
```
Comment (ardaguclu) [inline staging/src/k8s.io/kubectl/pkg/cmd/plugin/plugin.go:44]: I think, previous link is correct to install the Krew not installing the plugins.

Comment (Ritikaa96) [reply staging/src/k8s.io/kubectl/pkg/cmd/plugin/plugin.go:44]: ah! right. I added link seeing the previous line stating how to install plugins using krew

**Comment by Ritikaa96** [comment]
Hi @ardaguclu Thanks for pointing out the link, I have changed it back to original, Please let me know in case any other change is needed.

**Comment by ardaguclu** [comment]
Thank you
/approve
/lgtm
/triage accepted
/priority backlog

**Pull request merged**